cql-engine/src/main/java/org/opencds/cqf/cql/elm/execution/QueryEvaluator.java

```diff
@@ -256,8 +256,6 @@ protected Object internalEvaluate(Context context) {
             return null;
         }
 
-        context.clearLetExpressions();
-
```

Comment:
We still need to clear let expressions, right?

Reply:
No, they do not need to be cleared the way they are saved now.